src/assets/i18n/en.json

```diff
@@ -29,6 +29,7 @@
   "CREATE": "Create",
   "CUSTOM_NETWORK": {
     "ADDRESS_VERSION": "Address version",
+    "API_SERVER_PORT": "Seep Server API Port",
```

Comment:
Seed?

Reply:
Oops, didn't even notice 😂 thanks for fixing!